apps/consumer-client/pages/zuzalu-examples/sign-in.tsx

```diff
@@ -0,0 +1,115 @@
+import {
+  openSignedZuzaluSignInPopup,
+  SignInMessagePayload,
+  useFetchParticipant,
+  usePassportPopupMessages,
+  useSemaphoreSignatureProof,
+} from "@pcd/passport-interface";
+import { useEffect, useState } from "react";
+import { CollapsableCode, HomeLink } from "../../components/Core";
+import { ExampleContainer } from "../../components/ExamplePage";
+import { PASSPORT_SERVER_URL, PASSPORT_URL } from "../../src/constants";
+
+/**
+ * Example page which shows how to use a Zuzalu-specific prove screen to
+ * request a Semaphore Signature PCD containing the user's uuid as a third
+ * party developer.
+ */
+export default function Page() {
+  // We only do client-side proofs for Zuzalu UUID proofs, which means we can
+  // ignore any PendingPCDs that would result from server-side proving
+  const [pcdStr, _passportPendingPCDStr] = usePassportPopupMessages();
+
+  const [signatureProofValid, setSignatureProofValid] = useState<
+    boolean | undefined
+  >();
+  const onProofVerified = (valid: boolean) => {
+    setSignatureProofValid(valid);
+  };
+
+  const { signatureProof } = useSemaphoreSignatureProof(
+    pcdStr,
+    onProofVerified
+  );
+
+  // Extract UUID, the signed message of the returned PCD
+  const [signedMessage, setSignedMessage] = useState<
+    SignInMessagePayload | undefined
+  >();
+  useEffect(() => {
+    if (signatureProofValid && signatureProof) {
+      const signInPayload = JSON.parse(
+        signatureProof.claim.signedMessage
+      ) as SignInMessagePayload;
+      setSignedMessage(signInPayload);
+    }
+  }, [signatureProofValid, signatureProof]);
+
+  // Finally, once we have the UUID, fetch the participant data from Passport.
+  const { participant } = useFetchParticipant(
+    PASSPORT_SERVER_URL,
+    signedMessage?.uuid
+  );
+
+  return (
+    <>
+      <HomeLink />
+      <h2>Zuzalu UUID-revealing proof </h2>
+      <p>
+        This proof type is almost the same as <code>SempahoreSignaturePCD</code>
+        , except one key feature: the message that is 'signed' within this PCD
```

Comment:
`the message that is signed contains the user's unique identifier, or UUID. This lets you load the users's details from the Passport server, including their name, email and role.`

Reply:
OK - i added  a bunch more better copy on this page - see screenshot in PR description.